### PR TITLE
FM2-78 : Fix CIEL and LOINC concept sources empty result set  liquiba…

### PR DESCRIPTION
**Pull request by jecihjoy**
…se error

## Description of what I changed
Fixed CIEL and LOINC concept sources empty result set  liquibase error.

This issue was reported on talk https://bit.ly/2T3uq5W

## Issue I worked on

see https://issues.openmrs.org/browse/FM2-78

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**.

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -110,6 +110,32 @@
                                  referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
 
+	<changeSet id="add_loinc_openmrs_concept_source_20200220" author="jecihjoy" dbms="mysql">
```
Comment (corneliouzbett) [inline api/src/main/resources/liquibase.xml]: Rename to changeSet id to **`add_loinc_openmrs_concept_reference_source_20200220`**

File: api/src/main/resources/liquibase.xml [hunk 2]
```diff
@@ -110,6 +110,32 @@
...
+				<sqlCheck expectedResult="0">SELECT COUNT(*) FROM concept_reference_source WHERE name='LOINC' AND retired = 0</sqlCheck>
+			</and>
+		</preConditions>
+		<sql><![CDATA[
+			INSERT into concept_reference_source (name, description, hl7_code, creator, date_created, uuid)
+			VALUES ('LOINC', 'LOINC (www.loinc.org)', 'LN', 1, now(), '248b31c8-72fa-4b96-8d3d-b200dded985e')
+		]]></sql>
+	</changeSet>
+
+	<changeSet id="add_ciel_openmrs_concept_source_20200220" author="jecihjoy" dbms="mysql">
```
Comment (corneliouzbett) [inline api/src/main/resources/liquibase.xml]: rename changeSet id to **`add_ciel_openmrs_concept_reference_source_20200220`**

**Comment by ibacher** [comment]
I'd actually prefer it if we fixed this by changing the existing migrations to have `onFail="WARN"` and see if that addresses the issue. I don't really want to have the FHIR module managing non-FHIR related metadata. (Incidentally, this seems to be an issue with the version of Liquibase we're running in OpenMRS since `onFail="CONTINUE"` should... well, just continue rather than throwing an exception).

**Comment by corneliouzbett** [comment]
> I'd actually prefer it if we fixed this by changing the existing migrations to have `onFail="WARN"` and see if that addresses the issue. I don't really want to have the FHIR module managing non-FHIR related metadata. (Incidentally, this seems to be an issue with the version of Liquibase we're running in OpenMRS since `onFail="CONTINUE"` should... well, just continue rather than throwing an exception).

I agree with this

**Comment by jecihjoy** [comment]
Thanks @ibacher your comment made lot os sense. Also I've just updated the PR.
 The only I could make this work was by using **onError** attribute instead of **onFail**.

**Pull request merged**